tests/unit_tests/test_wave_06.py

```diff
@@ -193,8 +217,21 @@ def test_swap_best_by_category_no_match_is_false():
         my_priority="Clothing",
         their_priority="Clothing"
     )
+    assert result == False
+    assert len(tai.inventory) == 3
+    assert len(tai.inventory) == len(jesse.inventory)
```

Comment:
Line 222 works, but it will always be ever so slightly slower than checking `assert len(jesse.inventory) == 3`, since `len(tai.inventory) == len(jesse.inventory)` needs to evaluate `len` for both lists. Additionally, by checking the length of the lists against each other, it makes it seem like it's important to our function that `tai` and `jesse` have the same length inventories. Since that isn't the case, I suggest avoiding the comparison.